release-please-config.json

```diff
@@ -0,0 +1,45 @@
+{
+  "packages": {
+    "solutions/landing-zone": {
```

Comment:
Alain and I would like to have the "v" removed from the tag. it can be implemented for all packages using this block underneath line 2.
".": {
  "changelog-path": "CHANGELOG.MD",
  "include-v-in-tag": false
},

Reply:
@cartyc I don't see the block I suggested in the latest commit. Was it causing issues ?

Reply:
Ahh I see now. I just added it at the top of the JSON, line 4 and 5. Will revise in it's own block.

Reply:
well, I made a mistake and it won't work. you can revert to previous commit. sorry about that